src/content-script.ts

```diff
@@ -4,11 +4,16 @@ import Mustache from "mustache";
 import { createElement } from "./elements";
 import { isBodyWithReplies, isRepositoryDetails } from "./validations";
 
-(async function main() {
+document.addEventListener("soft-nav:end", main);
+
+async function main() {
```

Comment:
[Praise] This is looking great for running on soft navs - 💯!

[Request] One issue though, is that the `main` function no longer runs on the first page load. Which is unfortunate. If I add a `main()` call later on in the file it works again.

Reply:
Thanks for the catch! It's been a while